### ISSUE TITLE
[FEATURE] style dropdown properly

### ISSUE THREAD

**Issue by Dun-sin** [Dun-sin/Whisper#523]
### Description

replace this:
https://github.com/Dun-sin/Whisper/blob/335b5cd9ce9ad49b0158ccbff3809a600824533e/client/src/components/Anonymous.jsx#L207-L221

with this: 
```
<Dropdown placement="leftStart" style={{ zIndex: 3 }} renderToggle={MenuToggle} noCaret>
  <Dropdown.Item onClick={() => handleClose()} className='sm:w-[200px]'>
    <div className="flex justify-between gap-2 flex-wrap">
      <span>Close Chat</span>
      <span className="text-gray-500 text-xs">Ctrl + Shift + X</span>
    </div>
  </Dropdown.Item>
  <Dropdown.Item onClick={() => handleClose(true)} className='sm:w-[200px]'>
    <div className="flex items-center justify-between gap-2 flex-wrap">
      <span>Find a new buddy</span>
      <span className="text-gray-500 text-xs">Ctrl + Alt + N</span>
    </div>
  </Dropdown.Item>
</Dropdown>
```

### Are you willing to work on this issue ?

None

**Comment by Banerjee2021** [comment]
@Dun-sin Can you please assign me this issue?, I would like to work on it.

**Comment by Dun-sin** [comment]
> @Dun-sin Can you please assign me this issue?, I would like to work on it.

Definitely, thanks for wanting to contribute, read the issue description properly and understand what it tells you to do also feel free to ask questions, make sure you follow the rules [👉🏽here👈🏽](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md#rules), else your PR will not be accepted and will be closed. Good Luck.

**Comment by AviGawande** [comment]
if the issue is still not solved then i would like to work on it .

**Comment by Roshanchau** [comment]
Can you please assign me this issue? I would like to fix it.

**Comment by Dun-sin** [comment]
> Can you please assign me this issue? I would like to fix it.

Sure, thanks for wanting to contribute. Make sure to read the issue description carefully and ask if you have questions. Follow the rules [here](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md#rules), or your PR won't be accepted and will be closed. Good luck!
